docs/customization/template.rst

```diff
@@ -13,8 +13,23 @@ Sometimes even if you have decided to stay with the default layout provided by S
 business requirements**.
 You may just need to **add your logo anywhere**.
 
-How to customize templates?
----------------------------
+Methods of templates customizing
+--------------------------------
+
+.. warning::
+
+    There are two ways of customizing templates of Sylius:
+
+    The first one is simple **templates overriding** inside of the ``app/Resources`` directory of your project. Using
+    this method you can completely change the content of templates, it is a suggested way of integrating themes, custom
+    layouts.
+
+    The second method is **templates customization via events**. You are able to listen on these template events,
```

Comment:
Given the title of this section and the first line of the warning, isn't stating ``templates overriding/templates customization`` instead of simple ``overriding/customization`` a teeny tiny bit redundant?